build.go

```diff
@@ -71,12 +72,15 @@ func Build(entries EntryResolver, dependencies DependencyManager, planRefinery P
 		goLayer.Launch = entry.Metadata["launch"] == true
 
 		logs.Subprocess("Installing Go %s", dependency.Version)
-		then := clock.Now()
-		err = dependencies.Install(dependency, context.CNBPath, goLayer.Path)
+		duration, err := clock.Measure(
+			func() error {
+				return dependencies.Install(dependency, context.CNBPath, goLayer.Path)
+			},
+		)
```

Comment:
Nitpick: I'd tighten up this structure to look like this:

```suggestion
		duration, err := clock.Measure(func() error {
				return dependencies.Install(dependency, context.CNBPath, goLayer.Path)
		})
```